src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -1,12 +1,17 @@
 package mate.academy.service;
 
+import mate.academy.Main;
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(Main.class);
```

Comment:
Typically, the argument passed to getLogger() should be the class name of the class where the logger is used. It helps in distinguishing log messages based on the class name and provides better logging granularity. LogManager.getLogger(AuthenticationServiceImpl.class);